src/test/java/com/fasterxml/jackson/dataformat/xml/ser/TestSerialization.java

```diff
@@ -111,6 +111,16 @@ public void testSimpleAttrAndElem() throws IOException
         assertEquals("<AttrAndElem id=\"42\"><elem>whatever</elem></AttrAndElem>", xml);
     }
 
+    public void testNil() throws IOException
+    {
+        XmlMapper mapper = new XmlMapper();
+        mapper.configure(ToXmlGenerator.Feature.WRITE_NULLS_AS_XSI_NIL, true);
+        WrapperBean<String> bean = new WrapperBean<>(null);
+        // First, map in a general wrapper
+        String xml = mapper.writeValueAsString(bean);
+        assertEquals("<WrapperBean><value xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:nil=\"true\"/></WrapperBean>", xml);
```

Comment:
We can start with this, assuming it does match -- may need to use less exact matching to account for possible difference (underlying writer could produce empty element vs start/end and so on) but we can worry about that if that happens.

But I think it'd be good to have another separate test for property values, distinct from root value (like here)?

Reply:
Isn't it already a null property value (since `value` is null while WrapperBean is not)? Or am I misunderstanding what you're saying?

Reply:
Oh. Right you are, it IS a property value.

Conversely, then, testing root value would make sense. :)